### PR TITLE
Create Prison Visits prod namespace

### PR DESCRIPTION
**Pull request by KPobeeNorris**
Create Prison Visits Booking production namespace

### CODE REVIEW

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-visits-booking-production/resources/variables.tf [hunk 1]
```diff
@@ -0,0 +1,19 @@
+variable "environment-name" {
+  default = "production"
+}
+
+variable "team_name" {
+  default = "prison-visits-booking"
+}
+
+variable "is-production" {
```
Comment (digitalronin) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-visits-booking-production/resources/variables.tf]: Should this be "true" ?

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-visits-booking-production/resources/elasticache.tf [hunk 2]
```diff
@@ -0,0 +1,23 @@
+module "ec-cluster-prison-visits-booking-staff" {
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster?ref=3.0"
```
Comment (mtrblmoj) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-visits-booking-production/resources/elasticache.tf]: Could you please use the latest version of that module ? 3.1
https://github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster/releases/tag/3.1

**Comment by KPobeeNorris** [comment]
@digitalronin & @mtrblmoj - thanks for the review and apologies for missing the 'is-production" issue.  Changes have been pushed, so hopefully ok now :)

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-visits-booking-production/resources/elasticache.tf [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  cluster_name           = "${var.cluster_name}"
+  cluster_state_bucket   = "${var.cluster_state_bucket}"
+  team_name              = "${var.team_name}"
+  application            = "prison-visits-booking-staff"
```
Comment (mtrblmoj) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-visits-booking-production/resources/elasticache.tf]: Could you please add : 
```
node_type = cache.m4.xlarge
```

Comment (KPobeeNorris) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-visits-booking-production/resources/elasticache.tf]: @mtrblmoj - added

Comment (digitalronin) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-visits-booking-production/resources/elasticache.tf]: You've also got this failure in concourse: 

Error: module "ec-cluster-prison-visits-booking-staff": "aws_region" is not a valid argument

With this version of the module, you also need a providers block, as per:
https://github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster/blob/master/example/elasticache.tf

Comment (KPobeeNorris) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-visits-booking-production/resources/elasticache.tf]: @digitalronin - 🤞 the changes have worked

**Comment by KPobeeNorris** [comment]
@digitalronin - is it possible to request a final review?

**Pull request merged**